creme/drift/hddm_w.py

```diff
@@ -18,54 +18,59 @@ class HDDM_W(BaseDriftDetector):
     lambda_option : float (default=0.050)
         The weight given to recent data. Smaller values mean less weight given to recent data.
 
-    two_side_option : bool (default=True)
-        Option to monitor error increments and decrements (two-sided) or only increments (one-sided)
+    two_sided_test : bool (default=False)
+        If True, will monitor error increments and decrements (two-sided). By default will only
+        monitor error increments (one-sided).
 
     Notes
     -----
-    HDDM_W [1]_ is an online drift detection method based on McDiarmid's bounds. HDDM_W uses
+    HDDM_W is an online drift detection method based on McDiarmid's bounds. HDDM_W uses
     the EWMA statistic as estimator. It receives as input a stream of real predictions
     and returns the estimated status of the stream: STABLE, WARNING or DRIFT.
```

Comment:
I only figured that now, but the same applies to HDDM_A: do we need to change that since now only warning and drift indicators are returned? On the other hand, one could infer that `in_warning=False` and `in_drift=False` implies "stability". What do you think, @jacobmontiel?

Reply:
ADWIN and PageHinkley are the only methods that do not explicitly detect warning zones. For the rest, the 3 states are possible:

```
STABLE: in_warning=False, in_drift=False
WARNING: in_warning=True, in_drift=False
DRIFT: in_drift=True
```

In the case of DRIFT, `in_warning` becomes irrelevant.